base/multi.jl

```diff
@@ -1514,101 +1514,10 @@ macro everywhere(ex)
     end
 end
 
-function pmap_static(f, lsts...)
-    np = nprocs()
-    n = length(lsts[1])
-    Any[ remotecall(f, PGRP.workers[(i-1)%np+1].id, map(L->L[i], lsts)...) for i = 1:n ]
-end
-
+#FIXME delete?
```

Comment:
I'd remove the comment, and file an issue if you think this needs to be discussed/fixed.